File: Python/no. checking/co-prime.py
#Co prime no.
def prime(x):
    for i in range(2,x):
        if x%i==0:
            return False
    return True
n=int(input("Enter the 1st no. = "))
m=int(input("Enter the 2nd no. = "))
while m>0:
    r=n%m
    n=m
    m=r
if n==1 and prime(n) and prime(m):
    print("Co-Prime No.")
else:
    print("not Co Prime")

File: C Programms/No. Cheking In Range/co_prime.c
//WAP to check Co-Prime no.
#include<stdio.h>
void main()
{
    int a,b,f,r;
    printf("Enter The range : ");
    scanf("%d",&r);
    printf("Co-Prime no. in range of 1 - %d are ...\n",r);
    for(a=2;a<=r;a++)
    {
        for(b=2;b<=r;b++)
        {
            for(int i=1,f=1;i<=b;i++)
            {
                if(a%i==0 && b%i==0)
                    f=i;
            }
            if(f==1)
                printf("%d and %d \n",a,b);
        }
    }
}